jena-text/src/test/java/org/apache/jena/query/text/assembler/TestGenericAnalyzerAssembler.java

```diff
@@ -0,0 +1,198 @@
+package org.apache.jena.query.text.assembler;
```

Comment:
Missing an license header.  Please add it - it is better if the contributor does this than someone else, although the act of making a contribution is enough especially as other files have the header.